fvm/src/machine/manifest.rs

```diff
@@ -200,4 +212,9 @@ impl Manifest {
     pub fn get_embryo_code(&self) -> &Cid {
         &self.embryo_code
     }
+
+    /// Returns the code CID for the EOA actor.
```

Comment:
Can we explicitly call this the _ethereum_ EOA? Or just, Ethereum Account?

Reply:
Sure thing